### PR TITLE
Release/v0.11.2

### PR DESCRIPTION
**Pull request by rizakii**
## Bugfix
- Fix gateway address
- Fix base64 encode on gateway login

### CODE REVIEW

**Comment by rizakii** [comment]
@satoshikumano please review.

File: ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift [hunk 1]
```diff
@@ -67,13 +67,13 @@ public class GatewayAPI: NSObject, NSCoding {
             return
         }
 
-        let requestURL = "\(self.app.baseURL)/\(self.app.siteName)/token"
+        let requestURL = "\(self.gatewayAddress.absoluteString)/\(self.app.siteName)/token"
 
         // generate header
         let credential = "\(self.app.appID):\(self.app.appKey)"
         let base64Str = credential.dataUsingEncoding(NSUTF8StringEncoding)?.base64EncodedDataWithOptions(NSDataBase64EncodingOptions.Encoding64CharacterLineLength)
```
Comment (satoshikumano) [inline ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift]: Let's use
[NSData base64EncodedStringWithOptions]

Comment (rizakii) [inline ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift]: @satoshikumano 
```
[NSData base64EncodedStringWithOptions]
```
is used for decoding base64 string to plain nsdata.
In this case, we are encoding plain string into base 64 string.
I will rename the variable name to make it clear.

Comment (rizakii) [inline ThingIFSDK/ThingIFSDK/Gateway/GatewayAPI.swift]: hmm i got it, 

``` swift
let plainData = credential.dataUsingEncoding(NSUTF8StringEncoding)!
let base64Str = plainData.base64EncodedStringWithOptions(NSDataBase64EncodingOptions.init(rawValue: 0))!
```

**Comment by satoshikumano** [comment]
LGTM. please release it.

**Pull request merged**